libbeat/scripts/generate_template.py

```diff
@@ -156,15 +156,15 @@ def fill_field_properties(field, defaults, path):
         }
 
     elif field["type"] in ["geo_point", "date", "long", "integer",
-                           "double", "float", "keyword"]:
+                           "double", "float", "keyword", "boolean"]:
         properties[field["name"]] = {
             "type": field.get("type")
         }
         if field["type"] == "keyword":
             properties[field["name"]]["ignore_above"] = \
                 defaults.get("ignore_above", 1024)
 
-    elif field["type"] == "dict":
+    elif field["type"] in ["dict", "list"]:
```

Comment:
@tsg @monicasarbu Packetbeat had a type "list". I assumed this is identical to dict?